broker/src/main/java/io/camunda/zeebe/broker/bootstrap/PartitionManagerStep.java

```diff
@@ -33,7 +33,8 @@ void startupInternal(
             brokerStartupContext.getDiskSpaceUsageMonitor(),
             brokerStartupContext.getPartitionListeners(),
             brokerStartupContext.getCommandApiService(),
-            brokerStartupContext.getExporterRepository());
+            brokerStartupContext.getExporterRepository(),
+            brokerStartupContext.getCommandApiServerTransport());
```

Comment:
:thought_balloon: I've stumbled over this too and I really like your naming, "gatewayBrokerTransport". IMO we should rename things accordingly, starting with the method `getCommandApiServerTransport` and the class "AtomixServerTransport"

Reply:
:+1: I will create followup PR.